python/dgl/graphbolt/impl/torch_based_feature_store.py

```diff
@@ -560,28 +564,41 @@ class TorchBasedFeatureStore(BasicFeatureStore):
     ...     gb.OnDiskFeatureData(domain="node", type="paper", name="feat",
     ...         format="numpy", path="/tmp/node_feat.npy", in_memory=False),
     ... ]
-    >>> feature_sotre = gb.TorchBasedFeatureStore(feat_data)
+    >>> feature_store = gb.TorchBasedFeatureStore(feat_data)
```

Comment:
update this if you really want to add a new parameter.

Reply:
updated.